rules/config.js

```diff
@@ -56,6 +57,14 @@ module.exports = {
     'prefer-const': 2,
     'prefer-arrow-callback': 2,
     'strict': [ 2, 'never' ],
-    'no-const-assign': 2
+    'no-const-assign': 2,
+
+    // Node
+    'node/no-exports-assign': 'error',
+    'node/no-extraneous-require': 'error',
+    'node/no-missing-require': 'error',
+    'node/process-exit-as-throw': 'error', // This doesn't produce any errors
+    'node/no-deprecated-api': 'warn',
+    'node/shebang': 'warn'
```

Comment:
Do we put any node scripts in the `bin` for `package.json`? I get the impression we either invoke directly via `node $filepath` or in the scripts section instead.

Reply:
Not super often, but we do.